gui-e2e-testing/Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM mcr.microsoft.com/playwright:v1.45.3
+FROM --platform=linux/amd64 mendersoftware/mender-ci-tools:master AS mender-tools
```

Comment:
As discussed privately, this won't really achieve what you want.

We need to either update `mender-ci-tools:master` to use master versions of the tools or use the debian package as in [here](https://github.com/mendersoftware/create-artifact-worker/blob/master/Dockerfile#L17-L21), for example.